vendor/assets/leaflet/leaflet.osm.js

```diff
@@ -64,6 +64,16 @@ L.OSM.HOT = L.OSM.TileLayer.extend({
   }
 });
 
+L.OSM.GPS = L.OSM.TileLayer.extend({
+  options: {
+    url: document.location.protocol === 'https:' ?
+      'https://{s}.gps-tile.openstreetmap.org/lines/{z}/{x}/{y}.png' :
+      'http://{s}.gps-tile.openstreetmap.org/lines/{z}/{x}/{y}.png',
+    maxZoom: 20,
+    subdomains: 'abc'
+  }
+});
+
```

Comment:
Could it theoretically also be rolled up with the usage in https://github.com/openstreetmap/openstreetmap-website/pull/1406/files#diff-14dc1f5c2800c4a4d2accb07b96c97a0R59 though?